contracts/TopShot.cdc

```diff
@@ -922,6 +990,26 @@ pub contract TopShot: NonFungibleToken {
             return TopShot.currentSeries
         }
 
+        // createSubeditionResource creates new SubeditionMap resource that
+        // will be used to mint Moments with Subeditions
+        pub fun createSubeditionAdminResource() {
+            TopShot.account.save<@SubeditionAdmin>(<- create SubeditionAdmin(), to: /storage/TopShotSubeditionAdmin)
```

Comment:
Can this StoragePath be defined in a constant somewhere?

Reply:
I think the best we could do is a contract function that returns the path at this point (as we can't add fields to the contract). 
Not sure if:
```
 TopShot.account.save<@SubeditionAdmin>(<- create SubeditionAdmin(), to: Topshot.SubeditionAdminStoragePath())
 ```
 reads much better

Reply:
I also think it would be better to add this as a function so it can be used in transactions as well